src/UI/templates/default/Divider/divider.less

```diff
@@ -0,0 +1,13 @@
+/* not final */
+h6.il-divider {
+	padding: 3px 10px;
+	background-color: #f0f0f0;
```

Comment:
No color key in less files other than the variable.less, see: https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/templates/Readme.md#colors. Either use an existing color value directly or add an new one for dividers, e.g. divider-bg.